job-engine/commons/src/main/java/org/eclipse/kapua/job/engine/commons/operation/DefaultTargetReader.java

```diff
@@ -112,14 +117,22 @@ public Object readItem() throws Exception {
         JobLogger jobLogger = jobContextWrapper.getJobLogger();
         jobLogger.setClassLog(LOG);
 
-        jobLogger.info("Reading item...");
+        JobTargetQuery query = jobTargetFactory.newQuery(jobContextWrapper.getScopeId());
```

Comment:
I think this can be easier and can be done with the DeviceRegistryService.find(jobTarget.scopeId, jobTarget.tatgetId), like we now do on the `AbstractDeviceTargetProcessor`.